src/content/app/species/state/api/speciesApiTypes.ts

```diff
@@ -93,9 +93,12 @@ type VariationStatistics = {
   structural_variants_with_phenotype_assertions: number;
 };
 
-type RegulationStatistics = {
+export type RegulationStatistics = {
   enhancers: number;
   promoters: number;
+  ctcf_count: number;
+  tfbs_count: number;
+  open_chromatin_count: number;
```

Comment:
Notice the type defined in the api spec. These three added fields are nullable; therefore their type is `number | null`